carbonmark-api/src/models/Project.model.ts

```diff
@@ -29,6 +29,7 @@ export const ProjectModel = Type.Object({
       })
     )
   ),
+  serialization: Type.Optional(Type.String()),
```

Comment:
A convoluted answer for this one:

Long answer: This is the model creating a Project type using cms data, in this instance creating that same model from the ICR api which contains all data, not just project data. The tokenId used in other types comes from the subgraph queries. So the serialization was only needed here to create the url, which was then used as a lookup for the specific project and vintage for the individual project `DetailedProject`, which has tokenId.

Short answer: Now that the ICR projects are in the cms, there is no needed for the serialization or tokenId here and this will be removed in the next round.

Reply:
"and this will be removed in the next round."

Are we tracking this piece of work somewhere? @psparacino